lib/ansible/module_utils/basic.py

```diff
@@ -131,15 +131,15 @@
             break
     if algorithms is None:
         # python 2.5+
-        algorithms = ('md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512')
+        algorithms = ['md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512']
```

Comment:
The `getattr` a few lines above would also need to be cast to a list, as it returns a tuple by default:

```pycon
>>> hashlib.algorithms
('md5', 'sha1', 'sha224', 'sha256', 'sha384', 'sha512')
```